Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Driver.hs

```diff
@@ -1382,3 +1385,22 @@ data LinkRCWithDriverForFleetReq = LinkRCWithDriverForFleetReq
 
 instance HideSecrets LinkRCWithDriverForFleetReq where
   hideSecrets = identity
+
+----------- clear driver fee with create req ----
+
+type ClearDriverFeeAPI =
+  "clearDriverFee"
+    :> Capture "driverId" (Id Driver)
+    :> ReqBody '[JSON] ClearDriverFeeReq
+    :> Post '[JSON] APISuccess
+
+data ClearDriverFeeReq = ClearDriverFeeReq
+  { serviceName :: ServiceNames,
+    feeType :: DriverFeeType,
+    platformFee :: HighPrecMoney,
+    sgst :: HighPrecMoney,
```

Comment:
It should be calculated at backend instead of asking from UI.

Reply:
made it dynamic because there might be different gst rates for it.